Library/Formula/openjpeg.rb

```diff
@@ -22,7 +22,13 @@ class Openjpeg < Formula
   depends_on "libpng"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    system "cmake", ".", *args
+    system "make", "install"
```

Comment:
Yes, it first installs the dynamic build and then the static one. I don't think you can do them both simultaneously although I didn't try.